src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -570,8 +572,12 @@ public DevUtil(File buildDirectory, File serverDirectory, File sourceDirectory,
     }
 
     private void initGenerationContext() {
-        this.genContextDir = generateToSrc ? configDirectory : serverDirectory;
-        this.generatedFeaturesFile = new File(genContextDir, BinaryScannerUtil.GENERATED_FEATURES_FILE_PATH);
+        this.generatedFeaturesFileParent = generateToSrc ? configDirectory : generatedFeaturesTmpDir;
+        this.generatedFeaturesFile = new File(generatedFeaturesFileParent, BinaryScannerUtil.GENERATED_FEATURES_FILE_PATH);
+    }
+
+    public void copyTempFeatureFileToServer(File to) throws IOException {
+        copyFile(generatedFeaturesFile, generatedFeaturesFileParent, to, null);
```

Comment:
Do we need this method?

If so, we should probably rename to "copyFeatureFileToServer", since it will not always be the temp generated file, it could be generated in source.